components/channel_notifications_modal/components/expand_view.jsx

```diff
@@ -14,7 +14,7 @@ import SettingItemMax from 'components/setting_item_max.jsx';
 
 import Describe from './describe.tsx';
 import ExtraInfo from './extra_info.jsx';
-import SectionTitle from './section_title.jsx';
+import SectionTitle from './section_title.tsx';
```

Comment:
tsx ext is not needed

Reply:
@M-ZubairAhmed I think we cannot in JSX file
<img width="869" alt="Screenshot 2022-10-05 at 9 48 59 PM" src="https://user-images.githubusercontent.com/96474572/194110930-9c269a22-2d83-421c-b5b8-776c84a3e69d.png">

Reply:
i am not able to get that issue thought locally i tested, the path resolves file. Our webpack config supports checking for tsx first and then later jsx. https://webpack.js.org/configuration/resolve/#resolveextensions
May be its the issue in your code editors eslint.

Reply:
also apologies, i just pushed the commit to see if this error is caught in CI